generator/src/main/java/com/graphicsfuzz/generator/fuzzer/OpaqueExpressionGenerator.java

```diff
@@ -220,62 +216,42 @@ public OpaqueExpressionGenerator(IRandom generator, GenerationParams generationP
     // that, we make sure not to shift our 1 bit out of the integer by limiting our maximum shift
     // to 7 bits.
     final int minBitsForLowpUnsignedInt = minBitsForLowpInt - 1;
-    final int maxValue = generator.nextInt(isZero ? minBitsForLowpInt : minBitsForLowpUnsignedInt);
-    // We pass true as constContext when fuzzing here because the expression will be evaluated,
-    // so we don't want any side effects.
-    final Expr shiftValue = makeClampedFuzzedExpr(type, constContext, depth, fuzzer, maxValue);
```

Comment:
The naming of the various shiftValue variables might be a bit confusing - let me know if you think of better names for them, or if you'd rather me just comment what they're supposed to represent.